README.md

```diff
@@ -552,6 +552,17 @@ $ tx pull -f --mode=reviewed -l en,ar,es_419,fr,he,hi,ko_KR,pt_BR,ru,zh_CN
 
 [transifex-client]: https://docs.transifex.com/client/installing-the-client
 
+Javascript translations
+-----------------------
+
+The `djangojs.po` files created above are used to generate the `djangojs.js` files used
```

Comment:
Oh, I should have been clearer when I mentioned this was based on #156 as I cherry-picked those commits to build upon as we discussed in the ticket. The changes for this PR start on commit 4fd535f5799e316e8102d9aa3826ad0ace265839

I think we should wait on the other changes as @pomegranited will be working on some upstream comments for the original PR.

Reply:
I'm confused.. the domain used for XBlocks is `text`, not `djangojs`.  The configured [`STATICI18n_DOMAIN`](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/172/files#diff-3594922c93046c18881cbba04fc1e0deR76) is `text`, and all the translations live in `text.po` files, which are compiled to `text.mo` files, and generate `text.js` files, and the [XBlock Developer docs](https://edx.readthedocs.io/projects/xblock-tutorial/en/latest/edx_platform/edx_lms.html?highlight=translations#internationalization-support) confirm this.

The [README change from PR#156](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/156/files#diff-04c6e90faac2675aa89e2176d2eec7d8R558) uses `text` for this reason -- has this been changed?

Reply:
I had to change it to `djangojs` as that's the domain used in [django-admin makemessages](https://docs.djangoproject.com/en/1.11/ref/django-admin/#cmdoption-makemessages-domain) when extracting strings for translation.

I just renamed the reference in the README.md but now I see my message was pretty confusing in that aspect.

Reply:
I think you can pass `--domain text` to the `makemessage` command, and it will let you keep the `text` domain.  If we have to change the domain, then we have to change it for every single XBlock, and for the platform code that references the XBlock translations, so this is worth trying and preserving.

Reply:
Actually, I tried that but makemessages only supports domains `django` and `djangojs`

Reply:
Ok, then I doubt whether this is actually working, since the translation files are still `text.po` files.  But I'll have a closer look before merging this to my branch next sprint.

Reply:
Oh, I see now. Sorry for that, the commit that renamed the files was sitting on my machine.

The `djangojs` was configured [here](https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/172/files#diff-3594922c93046c18881cbba04fc1e0deR76) but I did not find a setting for the `django` domain. During testing it seemed to be picked automatically as the dummy translations were shown correctly. Am I doing this wrong?

Reply:
@viadanna , you need to make sure that the .js files you're generating actually have translated string in them.  For example, the original [ru/text.js](https://github.com/edx-solutions/xblock-drag-and-drop-v2/blob/997d625dbb7c0ef26684df1d5061a53d9a2f3b13/drag_and_drop_v2/public/js/translations/ru/text.js) has some translated strings, but the replacement one from this PR [ru/djangojs.js](https://github.com/edx-solutions/xblock-drag-and-drop-v2/blob/90f3bdc8e0b23bb930d7746139d60e22f9085083/drag_and_drop_v2/public/js/translations/ru/djangojs.js) does not.

Reply:
Dang, it seems the static translations aren't being generated at all for any of the languages, just the dummy one. I was pretty sure this worked during development, sorry for wasting your time here.

Reply:
Sorry, I don't have time right now to look at this, but I've set aside time next sprint, so will run and test it then.

Reply:
No worries, I'll take this up and make it work.